nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/ListS3.java

```diff
@@ -321,6 +322,8 @@ public class ListS3 extends AbstractS3Processor implements VerifiableProcessor {
     private volatile boolean justElectedPrimaryNode = false;
     private volatile boolean resetEntityTrackingState = false;
     private volatile ListedEntityTracker<ListableEntityWrapper<S3VersionSummary>> listedEntityTracker;
+    private Long minObjectAgeMilliseconds;
+    private Long maxObjectAgeMilliseconds;
```

Comment:
Please add `volatile` for the fields (like the others above). It is used because the variable is written and then read by different threads (written in `onScheduled`, read in `onTrigger`).